### PR TITLE
terminal: improve 'on' command

### PR DESCRIPTION
**Pull request by aarzilli**
Adds the ability to edit the list of commands executed after stopping
on a breakpoint, as well as converting a breakpoint into a tracepoint
and vice versa.

Prior to this it was possible to add commands to a breakpoint but
removing commands or changing a breakpoint into a tracepoint, or vice
versa, could only be done by removing and recreating the breakpoint.

### CODE REVIEW

File: pkg/terminal/command.go [hunk 1]
```diff
@@ -1700,43 +1710,52 @@ func breakpoints(t *Term, ctx callContext, args string) error {
...
+	}
+	if bp.Stacktrace > 0 {
+		attrs = append(attrs, fmt.Sprintf("%sstack %d", prefix, bp.Stacktrace))
+	}
+	if bp.Goroutine {
+		attrs = append(attrs, fmt.Sprintf("%sgoroutine", prefix))
+	}
+	if bp.LoadArgs != nil {
+		if *(bp.LoadArgs) == longLoadConfig {
+			attrs = append(attrs, "%sargs -v", prefix)
```
Comment (derekparker) [inline pkg/terminal/command.go:1738]: Needs `fmt.Sprintf`.

File: pkg/terminal/command.go [hunk 2]
```diff
@@ -1700,43 +1710,52 @@ func breakpoints(t *Term, ctx callContext, args string) error {
...
+		} else {
+			attrs = append(attrs, "%sargs", prefix)
```
Comment (derekparker) [inline pkg/terminal/command.go:1740]: Needs `fmt.Sprintf`.

File: pkg/terminal/command.go [hunk 3]
```diff
@@ -1700,43 +1710,52 @@ func breakpoints(t *Term, ctx callContext, args string) error {
...
+		}
+	}
+	if bp.LoadLocals != nil {
+		if *(bp.LoadLocals) == longLoadConfig {
+			attrs = append(attrs, "%slocals -v", prefix)
```
Comment (derekparker) [inline pkg/terminal/command.go:1745]: Needs `fmt.Sprintf`.

File: pkg/terminal/command.go [hunk 4]
```diff
@@ -1700,43 +1710,52 @@ func breakpoints(t *Term, ctx callContext, args string) error {
...
+		} else {
+			attrs = append(attrs, "%slocals", prefix)
```
Comment (derekparker) [inline pkg/terminal/command.go:1747]: Needs `fmt.Sprintf`.

**Review by derekparker — CHANGES_REQUESTED** [review]

**Comment by aarzilli** [comment]
Done.

**Review by derekparker — APPROVED** [review]
LGTM

**Pull request merged**